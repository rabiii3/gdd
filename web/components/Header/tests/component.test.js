import React from 'react';
import { shallow } from 'enzyme';
import { StyledHeaderLeft, StyledHeaderRight, StyledHeader } from '..';

describe('<HeaderLeft />', () => {
  const props = {
    children: <div />,
    classes: {},
  };
  test('should render a <StyledHeaderLeft />', () => {
    const tree = shallow(<StyledHeaderLeft {...props} />);
    expect(tree).toMatchSnapshot();
  });
});

describe('<HeaderRight />', () => {
  const props = {
    children: <div />,
    classes: {},
  };
  test('should render a <StyledHeaderRight />', () => {
    const tree = shallow(<StyledHeaderRight {...props} />);
    expect(tree).toMatchSnapshot();
  });
});

describe('<Header />', () => {
  const props = {
    children: <div />,
    classes: {},
  };
  test('should render a <StyledHeaderRight />', () => {
    const tree = shallow(
      <StyledHeader {...props}>
        <StyledHeaderLeft {...props} />
        <StyledHeaderRight {...props} />
      </StyledHeader>,
    );
    expect(tree).toMatchSnapshot();
  });
});
