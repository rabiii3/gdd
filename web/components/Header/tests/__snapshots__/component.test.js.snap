// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render a <StyledHeaderRight /> 1`] = `
<WithStyles(Grid)
  alignItems="center"
  container={true}
>
  <WithStyles(Grid)
    item={true}
    lg={7}
    sm={6}
    xs={8}
  />
  <WithStyles(Grid)
    item={true}
    lg={5}
    sm={6}
    xs={4}
  />
</WithStyles(Grid)>
`;

exports[`<HeaderLeft /> should render a <StyledHeaderLeft /> 1`] = `
<WithStyles(Grid)
  alignItems="center"
  container={true}
  direction="row"
  justify="flex-start"
  wrap="nowrap"
>
  <div />
</WithStyles(Grid)>
`;

exports[`<HeaderRight /> should render a <StyledHeaderRight /> 1`] = `
<WithStyles(Grid)
  alignItems="center"
  container={true}
  direction="row"
  justify="flex-end"
  wrap="nowrap"
>
  <div />
</WithStyles(Grid)>
`;
